packages/app-store/_utils/getCalendar.ts

```diff
@@ -19,8 +25,9 @@ export const getCalendar = async (credential: CredentialPayload | null): Promise
     return null;
   }
 
-  const calendarApp = await calendarAppImportFn();
-  if (!(calendarApp && "lib" in calendarApp && "CalendarService" in calendarApp.lib)) {
+  const calendarApp = (await calendarAppImportFn()) as CalendarApp;
+
+  if (!calendarApp.lib?.CalendarService) {
```

Comment:
I added this new `CalendarApp` type as I got this weird type error.

https://www.loom.com/share/7dd62f03e6af463bb51605408b0a9e8c

Reply:
we can solve this with a type guard as well

Reply:
Type narrowing to be exact: https://www.typescriptlang.org/docs/handbook/2/narrowing.html